drivers/hmis/spec/requests/hmis/graphql_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+require './drivers/hmis/spec/support/graphql_helpers'
+require 'rails_helper'
+
+RSpec.configure do |c|
+  c.include GraphqlHelpers
+end
```

Comment:
I think there is some rails magic that would lead me to not need this, but I can't figure it out

Reply:
I think we want to keep all this Graphql-related code within the driver, though.

Reply:
The driver inherits from the main app, so as long as `GraphqlHelpers` is in the main app or a gem, you can put it in the main `rails_helper.rb`

Reply:
Right but I think `GraphqlHelpers` should be in the driver, to keep all the HMIS-related code close together in the driver